### PR TITLE
Arc 1451 vera mart transactions daily rollup change transaction source from global message to everyaction

### PR DESCRIPTION
**Pull request by Frydafly**
# Pull Request for dbt-arc-functions

- [x] Create branch of dbt-arc-functions using JIRA extension so that naming convention is consistent

- [x] If you've created a macro, make sure it is documented by running the notebook: `create_docs.ipynb` util in this repo

- [x] Test branch on a development branch of an existing client (ideally the one that raised the bug). Do this by changing revision of the package to the branch name in `packages.yml` file in dbt State client below.
bsd-vera

- [x] If new macro files were created or updated, re-run `create_or_update_standard_models.py` for the client dbt project, replacing models and dependencies
**this change requires deletion of the staging schema in order for the project to run!**

- [x] check in dbt that client package successfully runs `dbt deps`, `dbt compile`, and `dbt run`; screenshot this page
Screenshot 2023-05-04 at 1 12 28 PM

- [x] query the resulting staging model in bigquery (or by previewing mart in dbt cloud, or your IDE); screenshot this
Screenshot 2023-05-04 at 1 26 34 PM

- [x] If exists, link bug issue and jira tickets to PR
relates to both https://bluestate.atlassian.net/browse/ARC-1450?atlOrigin=eyJpIjoiMzM3Nzg2Y2FjZmQ1NDc5NDgzODJkODU0ZjIzODc5YzciLCJwIjoiaiJ9 and ARC-1451

### CODE REVIEW

File: documentation/frakture_everyaction_transactions/staging/stg_frakture_everyaction_transactions.yml
Comment (ryantimjohn) [inline documentation/frakture_everyaction_transactions/staging/stg_frakture_everyaction_transactions.yml:1]: Blank doc

**Comment by ryantimjohn** [comment]
Got a blank doc but otherwise trust it because of https://github.com/bsd/arc-vera-dbt/pull/54

Can you update from main? Think it'll make the PR a lot easier to read.

**Comment by Frydafly** [comment]
@ryantimjohn branch was already up to date with main :/ 

I'll also go ahead and fix the blank doc, I was silly and completely forgot about installing codegen on vera's client project before running.

**Review by ryantimjohn — APPROVED** [review]
LGTM! And thanks for cleaning up these .MD files, I forgot to put that in my todos @Frydafly !

**Pull request merged**